Sources/XCTest/Private/PerformanceMeter.swift

```diff
@@ -11,7 +11,7 @@
 //  Measures the performance of a block of code and reports the results.
 //
 
-#if os(Linux) || os(FreeBSD)
+#if os(Linux) || os(FreeBSD) || os(iOS)
```

Comment:
As we discussed on Twitter, I think the way forward here isn't to hard-code Linux, FreeBSD, and now iOS to `import Foundation`. Instead, we should provide a configuration option of some kind, to compile swift-corelibs-xctest to use Foundation or SwiftFoundation, as the user desires.

To put it another way, I'm envisioning something like this:

```swift
#if defined(USE_SWIFT_FOUNDATION_XCODEPROJ)
    import SwiftFoundation
#else
    import Foundation
#endif
```

`USE_SWIFT_FOUNDATION_XCODEPROJ` would be specified by the user when they compile swift-corelibs-xctest, or as part of the SwiftXCTest-iOS scheme.